src/Core/Content/Test/Media/File/FileUrlValidatorMock.php

```diff
@@ -0,0 +1,24 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Test\Media\File;
+
+use Shopware\Core\Content\Media\File\FileUrlValidatorInterface;
+use Shopware\Core\Framework\Log\Package;
+
+/**
+ * @internal
+ */
+#[Package('discovery')]
+class FileUrlValidatorMock implements FileUrlValidatorInterface
+{
+    public function isValid(string $source): bool
+    {
+        $host = parse_url($source, \PHP_URL_HOST);
+
+        if ($host === false || $host === null) {
+            return false;
+        }
+
+        return true;
```

Comment:
there is a php validator for urls


```suggestion
        return filter_var($source, FILTER_VALIDATE_URL) !== false;
```

Reply:
The original behavior is that the host is used to validate the IP Address.



I have tested it with:
```php
$urls = [
    'test.com',
    'http://test.com',
];

foreach ($urls as $url) {
    $valid = filter_var($url, \FILTER_VALIDATE_URL) ? 'true' : 'false';

    var_export($url . ' - ' . $valid);
    echo '<br>';
}
```
This is the result
```
'test.com - false'
'http://test.com - true'
```

As we used a lot of URLs in tests without the protocol, **this could lead to new issues**

Reply:
ah ok, i see 👍